python/doc/examples/numerical_methods/optimization/plot_optimization_pagmo.py

```diff
@@ -0,0 +1,97 @@
+"""
+Multi-objective optimization using Pagmo
+========================================
+"""
+# %%
+# In this example we are going to explore optimization using OpenTURNS' `Pagmo <https://esa.github.io/pagmo2/>`_ interface.
+
+# %%
+from __future__ import print_function
+import openturns as ot
+from openturns.viewer import View
+ot.Log.Show(ot.Log.NONE)
+
+# %%
+# List available algorithms
+for algo in ot.Pagmo.GetAlgorithmNames():
+    print(algo)
+
+# %%
+# More details on Pagmo algorithms are available `here <https://esa.github.io/pagmo2/overview.html#list-of-algorithms>`_ .
+
+# %%
+# Create the problem, from ZDT test suite
+f = ot.SymbolicFunction(['x1', 'x2'], ['x1', 'var g := (1.0+9*x2); g * (1.0 - sqrt(x1 / g))'])
```

Comment:
I guess that your purpose is to reproduce this:
https://github.com/esa/pagmo/blob/80281d549c8f1b470e1489a5d37c8f06b2e429c0/src/problem/zdt.cpp#L159
From the code, I see a little difference in the way you define `g`, because the sum of inputs is missing (lines 168 to 170).The symbolic function should go like this:
```
'var s := x1 + x2; var g := 1 + 9 * s; g * (1 - sqrt(x1 / g))'
```
This may explain why the original picture of the Pareto front has a Y scale from 0 to 6, while the one you produce goes up to 0.8 only. 
Does it fit?

Reply:
yes, thanks